package.json

```diff
@@ -8,6 +8,7 @@
     "start-ui": "webpack-dev-server --open",
     "start": "npm run build-ui && node index.js",
     "build-ui": "webpack --config webpack.config.js",
+    "compile": "babel src -d lib",
```

Comment:
does this need to be ran in heroku ?

Reply:
would this need to be changed if it does?

Reply:
on a start we are building the ui would this need also be specified in the run to be compiled properly every time ? or does it just be run automatically

Reply:
I don't actually think this script is being used, will remove it.